doc/sphinx/practical-tools/utilities.rst

```diff
@@ -597,6 +599,8 @@ you to filter the list of modules, and ``(libraries
 the moment, Dune relies on Coq's standard mechanisms (such as
 ``COQPATH``) to locate installed Coq libraries.
 
+.. todo: COQPATH isn't explained anywhere; there is no example.  Does it still work?
```

Comment:
Yes, `COQPATH` works and is heavily relied on in the Nix setup. It is documented in the modules chapter, in one of the places you edited in this PR.

Reply:
The doc for that is inadequate: it is not complete nor easy to understand nor is there an example for this elementary bit of setup.  Hence my comment.  The format is the same as $PATH (colon separated on Linux, semicolon separated on Windows)?  And everything gets added to the Coq module?  And I have no idea about XDG* or how they would be used--are they platform specific?  Would also be nice to give some guidance about when to use COQPATH versus other mechanisms.

> the directories listed in the $COQPATH, ${XDG_DATA_HOME}/coq/ and ${XDG_DATA_DIRS}/coq/ environment variables (see XDG base directory specification) with the same physical-to-logical translation and with an empty logical prefix.

Reply:
I guess that the format is the same as `$PATH`. I'm not sure, but it's likely. A lot of programming language have a similar `$XXXPATH` environment variable.

>And everything gets added to the Coq module?

I'm not sure how it works exactly but basically it extends the places that Coq will treat like the `user-contrib/` directory in the Coq installation.